docs/packaging.md

```diff
@@ -41,13 +41,13 @@ the build command.
 When the self-update feature is disabled and a user tries to run `pixi self-update` an error message is displayed. This
 message can be customized by setting the `PIXI_SELF_UPDATE_DISABLED_MESSAGE` environment variable at build time to point
 the user to the package manager they should be using to update pixi.
-
-
+```shell
+PIXI_SELF_UPDATE_DISABLED_MESSAGE="`self-update` has been disabled for this build. Run `brew upgrade pixi` instead`" cargo build --locked --profile dist
```

Comment:
The annoying GitHub suggestions didn't let me add the final backticks of this code block.
Can you please add it manually?

Reply:
I think you might have somehow made this comment against an older version of this PR. The backticks are there in the most recent version and github marked your review comment as `Outdated`, despite there being no commits after your comment.